app/app.go

```diff
@@ -476,6 +481,9 @@ func NewRollapp(
 	// ... other modules keepers
 	tracer := cast.ToString(appOpts.Get(srvflags.EVMTracer))
 
+	app.RollappConsensusParamsKeeper = rollappparamskeeper.NewKeeper(
```

Comment:
inconsistent name between `RollappConsensusParamsKeeper` and RollappParamsKeeper?